### PR TITLE
ENG-7275 daily mediasoup-client 3.6.80 changes

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

**Comment by aconchillo** [comment]
I have added a couple of new commits to fix linting issues. Please, take a look.

File: src/handlers/Firefox117.ts [hunk 1]
```diff
@@ -0,0 +1,1093 @@
...
+		canvas.getContext('2d');
+
+		const fakeStream = (canvas as any).captureStream();
+		const fakeVideoTrack = fakeStream.getVideoTracks()[0];
+
+		try
+		{
+			pc.addTransceiver('audio', { direction: 'sendrecv' });
+
+			// START DEVIATION FROM FIREFOX 60
```
Comment (kompfner) [inline src/handlers/Firefox117.ts:113]: ah nice, you found these comments that i left! i just now remembered that i left these as "breadcrumbs" to reminder myself (or any reader) how we got the changes introduced in https://github.com/daily-co/pluot-core/pull/9291.

Comment (kompfner) [reply src/handlers/Firefox117.ts:113]: and https://github.com/daily-co/pluot-core/pull/9437

**Review by kompfner — APPROVED** [review]
Nice! LGTM, assuming you ran & validated the issues described in https://linear.app/dailyco/issue/CSDK-1686/firefox-117-remote-video-dead-bug and https://linear.app/dailyco/issue/CSDK-1698/when-using-firefox-after-muting-and-unmuting-your-camera-remote

File: src/handlers/Firefox117.ts (hunk 1 above)
Comment (aconchillo) [reply src/handlers/Firefox117.ts:113]: Yes, I left the comments because the file is exactly as Firefox60 except where those comments are.

**Comment by aconchillo** [comment]
> Nice! LGTM, assuming you ran & validated the issues described in https://linear.app/dailyco/issue/CSDK-1686/firefox-117-remote-video-dead-bug and https://linear.app/dailyco/issue/CSDK-1698/when-using-firefox-after-muting-and-unmuting-your-camera-remote

Yes, everything seems to keep working. For the second issue, this is a screenshot of the bitrate before and after turning the camera of from Firefox.

Screenshot 2024-03-14 at 6 47 50 PM

**Pull request merged**